site/components/pages/Resources/Container/index.tsx

```diff
@@ -0,0 +1,142 @@
+import Link from "next/link";
+import * as styles from "./styles";
+import { cx } from "@emotion/css";
+
+import {
+  PageWrap,
+  HeaderDesktop,
+  Footer,
+  NavItemDesktop,
+  HeaderMobile,
+  NavItemMobile,
+  ButtonPrimary,
+} from "@klimadao/lib/components";
+import { t, Trans } from "@lingui/macro";
+import ArrowBack from "@mui/icons-material/ArrowBack";
+
+import { urls } from "@klimadao/lib/constants";
+import { PageHead } from "components/PageHead";
+import { IS_PRODUCTION } from "lib/constants";
+import { FC, PropsWithChildren, ReactNode } from "react";
+
+type PageName = "blog" | "community" | "contact";
+
+export type Props = PropsWithChildren<ReactNode> & {
+  activePage: PageName;
+};
+
+export const Container: FC<Props> = (props) => {
+  const getListItemClasses = (name: PageName) => {
+    if (name === props.activePage) {
+      return cx(styles.listItem, styles.listItemActive);
+    }
+    return styles.listItem;
+  };
+
+  const _listItem = (name: PageName, textEl: ReactNode) => {
```

Comment:
Smart, but inline function-components are a pattern we should avoid because they cause a complete re-mount of children every time the parent re-renders. It's not a problem here, but could be a problem if someone refactors and adds a tooltip or a fancy dropdown or something.

I always prefer idiomatic HTML and CSS, even though it's more verbose:

```jsx
<Link href="/resources/contact">
  <a data-active={props.activePage === "contact"}>
    <Trans>Contact</Trans>
    <ArrowBack className="arrow" />
  </a>
</Link>
```


```css
a[data-active="true"] {
  /* active styles */
};
.arrow {
  display: none;
};
a[data-active="true"] .arrow {
  display: block;
};
```

Reply:
Yeah, much better. I have an aversion to data props for some reason so my brain goes sideways when it should be straightforward.